ecosystem/sep-0009.md

```diff
@@ -50,6 +50,7 @@ Name | Type | Description
 `bank_account_number` | string | Number identifying bank account
 `bank_number` | string | Number identifying bank in national banking system (routing number in US)
 `bank_phone_number` | string | Phone number with country code for bank
+`bank_branch` | string | Number identifying bank branch
```

Comment:
This field could be confused for a branch name, the fact it's a number is not clear from the name. Suggest using the format off some other fields here and using the `_number` suffix.

```suggestion
`bank_branch_number` | string | Number identifying bank branch
```